### PR TITLE
Athena/components

### PR DESCRIPTION
**Pull request by ath-chen**
created merchants page 
- added buttons to landing page but 100% open to changing, not sure how design wanted to stylize this so added as a placeholder

things to do: 
- update imageUrl with illustrations 
- bring in Bruce's progress bar & replace the one here
- update last donation made when/if this can be pulled from db

Screen Shot 2020-04-18 at 3 17 07 PM

Screen Shot 2020-04-21 at 9 58 31 PM

Screen Shot 2020-04-21 at 9 58 08 PM

### CODE REVIEW

**Comment by jtmckibb** [comment]
Yes locations exists on seller info, no, last donation made doesn't.

File: src/components/MerchantsPage/MerchantCard.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import styles from './styles.module.scss';
+import { SellerTwo } from '../../utilities/api';
+import temp from './temp.png';
+
+export interface Props {
+  storeInfo?: SellerTwo;
+}
+
+const MerchantCard = ({ storeInfo }: Props) => {
+  const storeName = storeInfo?.seller_id
```
Comment (jtmckibb) [inline src/components/MerchantsPage/MerchantCard.tsx:12]: I wouldn't want to rely on the URL to be the actual store name. eg) what if they have a char that doesn't work in URLs in their name such as Café China

Or even if there are multiple 99-favor-tastes, so we have to do 99-favor-taste-1 in the URL, even though it's called 99-favor-taste

File: src/components/MerchantsPage/MerchantCard.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+          <div className={styles.location}>Chinatown, Manhattan</div>
+          <h3> {storeName} </h3>
+          <p>
+            {' '}
+            100% of all gift card purchases and donations will go to support
+            this business and their staff.
+          </p>
+          <div style={{ color: '#949494' }}>
+            {/* need to fix this */}
+            Last donation 1h ago
```
Comment (jtmckibb) [inline src/components/MerchantsPage/MerchantCard.tsx:34]: @kevinjtsai This should be a different word, not donation. Last purchase? Last sent love?

File: src/utilities/api/types.ts [hunk 3]
```diff
@@ -36,3 +36,20 @@ export type Seller = {
   summary: string;
   story: string;
 };
+
+export type SellerTwo = {
```
Comment (jtmckibb) [inline src/utilities/api/types.ts:40]: What about BrowsePageSeller?

Comment (ath-chen) [reply src/utilities/api/types.ts:40]: Yes lol, much better

File: src/components/Home/styles.module.scss [hunk 4]
```diff
@@ -85,3 +86,28 @@
     height: 40px;
   }
 }
+
+.ctaButton2 {
```
Comment (jtmckibb) [inline src/components/Home/styles.module.scss:90]: primaryButton, secondaryButton?

File: src/components/MerchantsPage/MerchantCard.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      >
+        <div className={styles.merchantCard}>
+          <img className={styles.merchantCardLogo} src={temp} alt="Logo" />
+          <div className={styles.location}>
+            {city}, {state}
+          </div>
+          <h3> {storeInfo!.name} </h3>
+          <p>
+            {' '}
+            100% of all gift card purchases and donations will go to support
```
Comment (jtmckibb) [inline src/components/MerchantsPage/MerchantCard.tsx:35]: This should be the seller summary, ellipsified

File: src/components/MerchantsPage/MerchantsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  }, []);
+
+  return sellers ? (
+    <React.Fragment>
+      <div className={styles.container}>
+        <h2> Support our local merchants </h2>
+        <p>
+          Send Chinatown Love is intended to support our local businesses facing
+          financial loss. Make a difference today by donating or starting a
+          fundraiser.
```
Comment (jtmckibb) [inline src/components/MerchantsPage/MerchantsPage.tsx:29]: or buying a gift card

File: src/components/MerchantsPage/styles.module.scss [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+.location {
+  font-weight: bolder;
+  color: #46accc;
+}
+
+@media only screen and (max-width: 450px) {
+  .merchantCard {
+    width: 90vw;
```
Comment (jtmckibb) [inline src/components/MerchantsPage/styles.module.scss:43]: we should use percents here

**Pull request merged**